doc-Monitoring_Alerts_and_Reporting/topics/Operational_Alerts.adoc

```diff
@@ -297,7 +297,7 @@
 						
 					
 |
-
+							An influx of events from the Virtual Center or RHEV causes an inability of EVM/CFME to handle the capacity at which they are being queued. Utilize the Event Handler Configuration to filter events that are causing problematic queue table growth.
```

Comment:
We generally shouldn't abbreviate product names (though maybe this is different in tables)... however if we are OK to keep the abbreviation for "RHEV", it should be corrected to "RHV".